omnibus/config/software/system-probe.rb

```diff
@@ -0,0 +1,14 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-2020 Datadog, Inc.
+
+name 'system-probe'
+
+dependency 'libbcc'
```

Comment:
Should we comment this out for now until it's used? So we don't ship libbcc in 7.18 since it's not used.

Reply:
Good idea !
I’ve just commented the `libbcc` dependency out.